content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -444,9 +444,9 @@ The Automation beta includes the option to create a one-time email campaign, sim
 1. Select the **Export HTML** icon.
```

Comment:
On the Campaigns index page, hover over a campaign and select the **Export HTML** icon.

Reply:
I changed this up a little bit but did update this. Let me know what you think!